src/ui/public/directives/__tests__/paginated_selectable_list.js

```diff
@@ -68,6 +67,37 @@ lists.forEach(function (list) {
       expect(errorWrapper).to.throwError();
     }));
 
+    it('should throw an error with both makeUrl and onSelect attributes', function () {
```

Comment:
You can parameterize the `init` function and get rid of this duplication. Then you could just `expect(errorWrapper).to.throwError()`